dev-libs/libunistring/libunistring-1.3.recipe

```diff
@@ -66,13 +61,8 @@ INSTALL()
 	make install
 	rm -f "$libDir"/libunistring.la "$libDir"/charset.alias
 
-	if [ -n "$secondaryArchSuffix" ]; then
-		rm -rf "$documentationDir"
-		maybe_infoDir=
-	else
-		install -t "$developDocDir" README
-		maybe_infoDir="$infoDir"
-	fi
+	install -t $developDir/documentation/libunistring README
+	maybe_infoDir="$infoDir"
```

Comment:
This variable can be removed and replaced with `$infoDir` directly below.

Reply:
Wasn't sure here, will check up on that later.

Reply:
Updated, thanks!